draft-ietf-radext-radiusdtls-bis.md

```diff
@@ -338,6 +338,18 @@ In TLS-PSK operation at least the following parameters of the TLS connection sho
 
 [^pskid]: TODO: What is the correct term here? "PSK Identifier"? Probably not "TLS Identifier" as it was in RFC6614
 
+## Server Identity
+
+For clients it may be important to uniquely identify a server, i.e. to decide whether all connections to this server are down or if there is already an active connection to a server that was discovered using dynamic discovery described in {{RFC7585}}.
+In RADIUS/UDP a server identity could be simply described as the tuple of destination IP address and destination port.
+
+For RADIUS/(D)TLS, this distinction is not that simple, i.e. because different RADIUS servers are hidden behind a load balancer with one public IP address, and the load balancer dispatches the connections based on the Server Name Indication in the first TLS handshake record.
```

Comment:
```suggestion
For RADIUS/(D)TLS, this distinction is not that simple, i.e. because different RADIUS servers may be hidden behind a load balancer with one public IP address, and the load balancer dispatches the connections based on the Server Name Indication in the first TLS handshake record.
```